pkgs/tools/security/pinentry/mac/.editorconfig

```diff
@@ -0,0 +1 @@
+root = true
```

Comment:
This is here to stop editorconfig from throwing errors on binary nib files.

Reply:
Please move this into the top-level file

Reply:
Done.